sortinghat/core/utils.py

```diff
@@ -119,3 +120,32 @@ def unaccent_string(unistr):
     string = ''.join(cs)
 
     return string
+
+
+def validate_field(name, value, allow_none=False):
+    """Validate a given string field following a set of rules
+
+    The conditions to validate `value` consists on checking if its value is `None`
+    and if this is allowed or not; if its value is not an empty string or if it is
+    not composed only by whitespaces and/or tabs.
+
+    :param name: Name of the field to validate
+    :param value: Value of the field to validate
+    :param allow_none: `True` if `None` values are permitted, `False` otherwise
+
+    :raises ValueError: when a condition to validate the string is not satisfied
+    :raises TypeError: when the input value is not a string and not `None`
+    """
+    is_string = isinstance(value, str)
```

Comment:
Probably the first thing is to compare with `None` cases, so you don't have to add extra checks all over the code.

Reply:
Done.